packages/manager/apps/key-management-system/src/api/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { OKMS } from '@/interface';
+
+export const defaultCompareFunction = (key: keyof OKMS) => (
```

Comment:
Do you really need to create a generic compare function whereas you only use it with `id` key ? Moreover, it won't work with `iam` key which is not a `string`